sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/RecognizePiiEntitiesActionResult.java

```diff
@@ -5,16 +5,74 @@
 
 import com.azure.ai.textanalytics.implementation.RecognizePiiEntitiesActionResultPropertiesHelper;
 import com.azure.ai.textanalytics.util.RecognizePiiEntitiesResultCollection;
+import com.azure.core.util.logging.ClientLogger;
+
+import java.time.OffsetDateTime;
+
+import static com.azure.ai.textanalytics.implementation.Utility.throwExceptionIfError;
 
 /**
  * The {@link RecognizePiiEntitiesActionResult} model.
  */
-public final class RecognizePiiEntitiesActionResult extends TextAnalyticsActionResult {
+public final class RecognizePiiEntitiesActionResult {
```

Comment:
@maririos  This API changes will affect .NET API as well. .NET has the class TextAnalyticsActionDetails. 
https://apiview.dev/Assemblies/Review/b196a5e53fa2441f92e3b1accefa35e7#Azure.AI.TextAnalytics.TextAnalyticsActionDetails

Reply:
At least for .NET, I don't fully agree with the change though. We already have a `TextAnalyticsResult` class that is used by all of the sync endpoints and has the same information. It does not have a public constructor so users can't instantiate it and help us avoid duplicated code. Looking at your PR, the fact that your class is abstract means that in every `xxAction` class you need to implement at least 4 properties.

Also, note that for the last preview, as a team we agreed to rename the class to `TextAnalyticsActionDetails`.

All of that said, if Java team thinks this is the best design for Java, I have no objections to it.

Reply:
Trying to layout the tradeoff here for the overview:
(1) Follow what already GAed pattern in `TextAnalyticsResult`, which keep the `TextAnalyticsActionResult` but (might be) rename to `TextAnalyticsActionDetails`.

Pros: Same pattern as 'TextAnalyticsResult`, which all common properties can be shared in all its derived class.
Cons:  This class `TextAnalyticsActionResult` is never used directly by users. so there is a cost of maintaining the class.
 
(2) Remove `TextAnalyticsActionResult` and duplicate all the properties to the derived class.

Pros: `TextAnalyticsActionResult` is never used directly by users. so we will reduce the complexity of inheritance and without introduce a new class.
Cons: it might looks like it break up the pattern in TextAnalyticsResult, since both TextAnalyticsResult and TextAnalyticsActionResult could be an error result or a successfully completed result.

@srnagar